webapp/src/tutorial.tsx

```diff
@@ -288,7 +288,7 @@ export class TutorialHint extends data.Component<ISettingsProps, TutorialHintSta
         } else {
             let onClick = tutorialStep < tutorialStepInfo.length - 1 ? this.next : this.closeHint;
             const actions: sui.ModalButton[] = [{
-                label: lf("Start"),
+                label: lf("Ok"),
```

Comment:
Did we want this to be start for hoc still? / should this be `Start` when in hideiteration + flyoutonly?

Reply:
oh sure sure good point! I'll fix